### ISSUE TITLE
create profile card

### ISSUE THREAD

**Issue by AVP4000** [nss-evening-cohort-27/gitsub-doing-buzinezz-azz#6]
card container should have the following -

1. Profile picture
2. Full Name
3. Preferred Name
4. About me
5. Follow, Sponsor button etc
6. number of followers, # of following, # of favorited
7. Business Location: ie. NYC in NY
8. Contacts: Email Address ie. github, personal website. twitter ie. other social medias
9. Highlights of the user's work experience ie. Github Star, git hub pro user, arctic code vault contributor
10.  Organizations/Groups that the user is a part of ie. Cohort 27, Nashville Data Engineering Group, Nashville .NET User Group
11. Sponsors - other users and organizations that provide credibility or are familiar with the user
12.  Card that encompasses everything

**Stretch Goal**
Try to get these other elements clickable or adding favorites, but only after you have completed the MVP as a team.

![image](https://github.com/nss-evening-cohort-27/gitsub-doing-buzinezz-azz/assets/161537912/3e7a3733-3a54-4693-aec4-e2b03842fd27)
